lib/analytics/analytics_servce.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:ax_dapp/analytics/event.dart';
+
+class AnalyticsService {
```

Comment:
We should move away from the concept of services, since they are not part of the new architecture.